table.cc

```diff
@@ -2441,7 +2494,8 @@ table::make_reader_excluding_sstable(schema_ptr s,
     auto effective_sstables = ::make_lw_shared<sstables::sstable_set>(*_sstables);
     effective_sstables->erase(sst);
 
-    readers.emplace_back(make_sstable_reader(s, std::move(effective_sstables), range, slice, pc, std::move(trace_state), fwd, fwd_mr));
+    // no resource tracking as this there is just one reader like this
```

Comment:
Not sure why this is a reason not to track it. This reader still consumes resources. If we have many sstables, it can consume a lot of memory.

Reply:
Agree it's safer. @psarna opinion?

Reply:
I'm not 100% sure what "resource tracking" is, but if it's taking the resources into account in order to limit resource usage, then I agree it should be taken into account. Currently the view update generator serves the sstables one by one (per shard) and limits its max queue size to `5` with a semaphore, but it sounds more future-proof to add tracking now.

Reply:
I did so in the latest version. Yes, resource tracking tracks how much memory is used by the reader and is used to queue new readers until the older ones retire and release their memory,